std/stdio.d

```diff
@@ -3258,14 +3258,20 @@ struct LockingTextReader
 }
 
 /**
- * Indicates whether $(D T) is a file handle of some kind.
+ * Indicates whether $(D T) is a file handle, i.e. the type
+ * is implicitly convertable to $(LREF File) or a pointer to a
+ * $(REF FILE, core,stdc,stdio).
+ *
+ * Returns:
+ *      `bool`
```

Comment:
My (personal) stance on this is that if the function can be completely described by the `Returns:` section, then the `Returns:` section can simply *be* the function description. I.e., just omit the main description text.